diff -ru b/gamescope-session-script a/gamescope-session-script
--- b/gamescope-session-script	2023-04-27 17:22:14.428995862 +0300
+++ a/gamescope-session-script	2023-04-27 17:21:30.549189166 +0300
@@ -7,6 +7,13 @@
 export INTEL_DEBUG=norbc
 export mesa_glthread=true
 
+# Set Gamescope args
+export GAMESCOPE_EXTRA_LAUNCH_ARGS='--hdr-enabled --hdr-itm-enable'
+export GAMESCOPE_BASE_LAUNCH_ARGS='--xwayland-count 2 -O *,eDP-1 --default-touch-mode 4 --hide-cursor-delay 3000 --fade-out-duration 200':$GAMESCOPE_EXTRA_LAUNCH_ARGS
+
+export ENABLE_GAMESCOPE_WSI=1 
+export DXVK_HDR=1
+
 # Some environment variables by default (taken from Deck session)
 export SDL_VIDEO_MINIMIZE_ON_FOCUS_LOSS=0
 
@@ -184,22 +191,14 @@
         $CURSOR \
         -e \
         $RESOLUTION \
-        --xwayland-count 2 \
-        -O *,eDP-1 \
-        --default-touch-mode 4 \
-        --hide-cursor-delay 3000 \
-        --fade-out-duration 200 \
+        $GAMESCOPE_EXTRA_LAUNCH_ARGS \
         -R $socket -T $stats"
      else
         GAMESCOPECMD="/usr/bin/gamescope \
         $CURSOR \
         -e \
         $RESOLUTION \
-        --xwayland-count 2 \
-        -O *,eDP-1 \
-        --default-touch-mode 4 \
-        --hide-cursor-delay 3000 \
-        --fade-out-duration 200 \
+        $GAMESCOPE_EXTRA_LAUNCH_ARGS \
         --prefer-vk-device $VULKAN_ADAPTER \
         -R $socket -T $stats"
      fi
